src/hooks/useBoosts.ts

```diff
@@ -11,14 +11,14 @@ type Props = {
 
 export function useBoosts({ userAddress, gameId }: Props) {
   const { data, isLoading, error } = useQuery<Boost[]>(
-    ['boosts', userAddress, gameId],
+    ['boosts'],
```

Comment:
this is the cache key generation. Not a big deal since gameID and userAddress won't change but if they were to log out and sign back in with different wallet, it might pull data from previous address. Let's add them back in

Reply:
makes sense here